### ISSUE TITLE
Payara server node and output view do not refresh in NetBeans 9

### ISSUE THREAD

**Issue by johbor** [payara/ecosystem-netbeans-plugin#7]
I installed NetBeans 9.0 on my Mac (10.12.6) and run it with JDK9.
Configured 3 servers, Tomcat 9.10, GlashFish web 5.0.1 and Payara web 5.182
Tomcat starts normal in via the servers node and shows the output logs. Expanding the node shows the web apps.
But starting GlassFish and Payara doen’t show the output logs and the node stays on the hourglass and is not expandable.
Payara is started with JDK8.

I waited a little while also after testing with Payara 5.183 but then suddenly a small icon appears down to bottom of NB.
There is an error.
It is complaining about a missing resource.

`
INFO [payara]: Storing Payara web 5.182 atributes
INFO [payara]: Created Payara Server 5.182 instance with name Payara web 5.183
WARNING [payara-javaee]: Warning: Java EE documentation not found when registering EclipseLink library.
INFO [payara-javaee]: Java EE documentation not found when registering Java EE API library.
INFO [payara]: Storing Payara web 5.183 atributes
INFO [null]: Last record repeated again.
SEVERE [global]
java.util.MissingResourceException: Can't find bundle for base name sun.util.logging.resources.logging, locale en_US
	at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2045)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1679)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1583)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1546)
	at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:914)
	at org.netbeans.modules.payara.common.LogViewMgr.getLocalized(LogViewMgr.java:317)
	at org.netbeans.modules.payara.common.LogViewMgr.<init>(LogViewMgr.java:299)
	at org.netbeans.modules.payara.common.LogViewMgr.getInstance(LogViewMgr.java:172)
	at org.netbeans.modules.payara.common.LogViewMgr.displayOutput(LogViewMgr.java:1091)
	at org.netbeans.modules.payara.common.actions.ViewServerLogAction.performAction(ViewServerLogAction.java:76)
	at org.openide.util.actions.NodeAction$DelegateAction$1.run(NodeAction.java:561)
`

I saw this earlier and it was complaining about local en_GB.
So I changed the startup of NetBeans in netbeans.conf by adding 2 parameters to netbeans_default_options
`
-J-Duser.language=en -J-Duser.country=US
`

I thought maybe the the redraw of the node is not done because it is waiting for the search result of the resource.

[log-Start-NB-Payara-2.txt](https://github.com/payara/ecosystem-netbeans-plugin/files/2356143/log-Start-NB-Payara-2.txt)
[IDE.log](https://github.com/payara/ecosystem-netbeans-plugin/files/2356144/IDE.log)
[payara-start-server.log](https://github.com/payara/ecosystem-netbeans-plugin/files/2356145/payara-start-server.log)
![screen shot 2018-09-06 at 10 16 41](https://user-images.githubusercontent.com/9353297/45144307-0d4f3f00-b1be-11e8-9ab7-5f41469d2766.png)

**Comment by jGauravGupta** [comment]
Thanks @johbor for reporting the issue.

Internal ticket : NP-7

**Comment by johbor** [comment]
Tested the issue also with the 1.4 plugins because the report was done on 1.3 but the same results in not expanding the server node and not showing the output view of Payara log.

**Comment by jGauravGupta** [comment]
May you also confirm this issue with JDK 8 ?

**Comment by johbor** [comment]
Payara is started in both cases of the 1.3 and 1.4 plugin with JDK8.
NetBeans 9 runs on JDK9.
![screen shot 2018-09-06 at 12 08 40](https://user-images.githubusercontent.com/9353297/45151094-b5b8cf80-b1cd-11e8-868a-5ab2cc704f31.png)
![screen shot 2018-09-06 at 12 09 00](https://user-images.githubusercontent.com/9353297/45151095-b5b8cf80-b1cd-11e8-9530-8b91af95ea75.png)

**Comment by shivinm** [comment]
I am seeing this issue on jdk 8 and payara 5.183 when trying to start Payara from eclipse. Any idea when this will be fixed? Is there a workaround for it? 

@jGauravGupta 
<img width="899" alt="screen shot 2018-11-26 at 11 47 42 am" src="https://user-images.githubusercontent.com/5727055/49035171-3950df80-f171-11e8-9735-5c5da57842a8.png">

**Comment by jGauravGupta** [comment]
Hi @shivinm,
For issue related to Eclipse IDE plugin, please report it here : https://github.com/payara/ecosystem-eclipse-glassfish-plugin/issues

**Comment by HipDragon** [comment]
I tried it on Netbeans 9, JDK 10, and Payara Plugin 1.5 and it still had the same problem. I tried it with Netbeans 9, JDK 8, Payara Plugin 1.5 and can start and stop the server now but am unable to run or debug a project to the server. It recompiles the source each time and says it's going to start the server but hangs there. Tried starting the server then running and debugging the project same results.
